src/App.js

```diff
@@ -1,29 +1,4 @@
-import logo from './logo.svg';
 import './App.css';
+import mainCalculator from './components/calculator';
 
```

Comment:
- [ ] Similarly, let's make the main component, _App_ component a class-based component. The _App_ component should be similar to:

```javascript
// src/App.js

import React, { Component } from 'react';
import Calculator from ./components; // Import Calculator component

class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```